test/markup/yaml/complex-keys.txt

```diff
@@ -1,18 +1,13 @@
-"*":
-  "**/.env"
+"**/.env"
 
-"octo-repo":
```

Comment:
Why did we lose all the keys?

Reply:
@joshgoebel It wasn't a key value that needed to be tested, but a title for that test😂. I was mistaken at first and added that part to the test.

Reply:
@joshgoebel It wasn't a key value that needed to be tested, but a title for that test😂. I was mistaken at first and added that part to the test.